csharp/serialization/Loan.cs

```diff
@@ -33,18 +28,15 @@ public string Customer
             }
         }
 
-        // <Snippet3>
-        [field: NonSerialized()]
-        public event System.ComponentModel.PropertyChangedEventHandler PropertyChanged;
-        // </Snippet3>
+        public event PropertyChangedEventHandler? PropertyChanged;
 
         public Loan(double loanAmount,
                     double interestRate,
                     int term,
                     string customer)
         {
             this.LoanAmount = loanAmount;
-            this.InterestRatePercent = interestRate;
+            this.InterestRate = interestRate;
```

Comment:
As part of these changes could you also please update the name of the `customer` field to `_customer`? That way we can remove all of the redundant `this.` qualifiers.